sdk/src/Core/Amazon.Runtime/EventStreams/IEventStreamEvent.cs

```diff
@@ -15,7 +15,7 @@
 
 using System.Diagnostics.CodeAnalysis;
 
-namespace Amazon.Runtime.EventStreams.Internal
+namespace Amazon.Runtime.EventStreams
```

Comment:
I moved `IEventStreamEvent` to `Amazon.Runtime.EventStreams` out of the `Internal` namespace because we expose this interface as part of the public API. When you enumerate or async enumerate the events from the response the object return in the enum is of type `IEventStreamEvent`. This is a bad behavior of v3 that I wanted to fix in v4.